### PR TITLE
Fixes all current warnings

### PR DESCRIPTION
**Pull request by Pandabear1125**

### CODE REVIEW

File: src/sensors/ICM20649.cpp [hunk 1]
```diff
@@ -13,13 +13,16 @@ void ICM20649::init(CommunicationProtocol protocol)
     switch (protocol)
     {
     case I2C:
+    {
```
Comment (StudentKyle) [inline src/sensors/ICM20649.cpp:16]: brackets not consistent with other PR

Comment (Pandabear1125) [reply src/sensors/ICM20649.cpp:16]: This fixes a cross initialization. This is intended

Comment (StudentKyle) [reply src/sensors/ICM20649.cpp:16]: can the brackets be made on the same line? Either work to create scope here but just to keep it consistent with the rest of the formatting you just did

Comment (StudentKyle) [reply src/sensors/ICM20649.cpp:14]: Unless you think it's valuable to be different here since it is to fix an unclear error

Comment (StudentKyle) [reply src/sensors/ICM20649.cpp:14]: Then it's fine how you have it, it being different here could actually be helpful

**Review by Rishi-H — APPROVED** [review]

**Pull request merged**